packages/demo/components/pages/apis/transaction/builder/scriptTxIn.tsx

```diff
@@ -5,7 +5,7 @@ export default function ScriptTxIn() {
   return (
     <Section
       sidebarTo="scriptTxIn"
-      header="Set script input for transaction"
+      header="Use a script as input for a transaction"
```

Comment:
Same as above

Reply:
Same as before. My suggestions are based on grammar and clarity, leaning more towards explicit description, while it seems your preference is for "short, snappy and blunt" imperatives. In some sense it is a style-choice, so I'm fine as you prefer it.